scalatest/src/main/scala/org/scalatest/fixture/AsyncWordSpecLike.scala

```diff
@@ -50,7 +50,7 @@ StringVerbBlockRegistration, SubjectWithAfterWordRegistration}
  */
 //SCALATESTJS-ONLY @scala.scalajs.js.annotation.JSExportDescendentClasses(ignoreInvalidDescendants = true)
 @Finders(Array("org.scalatest.finders.WordSpecFinder"))
-trait AsyncWordSpecLike extends AsyncTestSuite with AsyncTestRegistration with ShouldVerb with MustVerb with CanVerb with Informing with Notifying with Alerting with Documenting { thisSuite =>
+trait AsyncWordSpecLike extends AsyncTestSuite with AsyncTestRegistration with ShouldVerb[Future[PendingStatement]] with MustVerb[Future[PendingStatement]] with CanVerb[Future[PendingStatement]] with Informing with Notifying with Alerting with Documenting { thisSuite =>
 
```

Comment:
I think we'll need to make an AsyncShouldVerb, AsyncMustVerb, and AsyncCanVerb where the signature of is has Future[PendingStatement], else this is a breaking change and couldn't go into a 3.x.y release. Plus it makes the types a bit simpler (at the cost of more classes).